docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ReplaceMissingValues.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.ML.Data;
+using Microsoft.ML.SamplesUtils;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class ReplaceMissingValues
+    {
+        public static void Example()
+        {
+            // Creating the ML.Net IHostEnvironment object, needed for the pipeline.
+            var mlContext = new MLContext();
+
+            // Download the training and validation files.
+            string dataFile = DatasetUtils.DownloadMslrWeb10k();
```

Comment:
>DownloadMslrWeb10k [](start = 43, length = 18)

You can use the `LoadFeaturizedMslrWeb10kDataset` instead which will save you creating the data processing step. This is the standard in the samples.

Reply:
My feeling is that we should use in-memory data when possible. In, #2726, we discuss some of in-memory data's advantages.